asllib/Interpreter.ml

```diff
@@ -1036,9 +1037,10 @@ module Make (B : Backend.S) (C : Config) = struct
               vs
           else (
             List.map B.debug_value vs |> String.concat " " |> print_string;
```

Comment:
Remove the `" "` from `String.concat` - there should be no spaces between the individual strings.